release-notes/v5.5.0.md

```diff
@@ -0,0 +1,113 @@
+
+#### <sub><sup><a name="v550-note-4" href="#v550-note-4">:link:</a></sup></sub> feature
+* We noticed after #4058 (where build steps are collapsed by default) that it wasn't very easy to see failing steps.
+  Now a failing step has a red border around its header, an erroring step has an orange border, and a running step has a yellow border. #4164, #4250
+
+#### <sub><sup><a name="v550-note-5" href="#v550-note-5">:link:</a></sup></sub> feature
+
+* On particularly busy clusters, users have observed [metrics events](https://github.com/concourse/concourse/issues/3674) [being dropped](https://github.com/concourse/concourse/issues/3769) due to a full queue #3937. @rudolfv added a configurable buffer size for metrics emission, regardless of your configured emitter. This should allow operators to trade memory pressure on the web nodes for reliability of metric transmission.
+
+#### <sub><sup><a name="v550-note-6" href="#v550-note-6">:link:</a></sup></sub> feature
+
+* @rudolfv also added some features for the special case of InfluxDB metrics. To decrease the request load on InfluxDB, you can configure the number of events to batch into a single InfluxDB request, or you can specify a hardcoded interval at which to emit events, regardless of how many have accumulated #3937.
+
+#### <sub><sup><a name="v550-note-7" href="#v550-note-7">:link:</a></sup></sub> fix
+
+* @evanchaoli improved `fly` - when outputting sample commands to the terminal, the CLI is aware of the path from which it is being executed #4284.
+
+#### <sub><sup><a name="v550-note-8" href="#v550-note-8">:link:</a></sup></sub> fix
+
+* The web UI used to [silently break](https://github.com/concourse/concourse/issues/3141) when your token (which includes a potentially-long JSON-encoded string detailing all the teams you are part of and what roles you have on them) was longer than the size of a single cookie (4096 bytes on most browsers!). This limit has been increased 15-fold, which should unblock most users on clusters with a lot of teams #4280.
+
+#### <sub><sup><a name="v550-note-9" href="#v550-note-9">:link:</a></sup></sub> fix
+
+* For the past few releases, the web nodes have allowed themselves to make up to 64 parallel connections to the database, to allow for parallelizing work like GC and scheduling within a single node. @ebilling has configured the web node's tolerance for idle connections to be more lenient: If a node has been using more than 32 of its available connections, up to 32 connections will be allowed to stay idly open. Anecdotally, CPU savings (resulting from less opening/closing of connections) of up to 30% have been observed on web nodes because of this change. Furthermore, the total max connection pool size has been made configurable - this should allow operators to avoid overloading the max connection limit on the database side #4232.
```

Comment:
this was indeed great! in `hush-house` (where we use TLS) we observed CPU utilization reduction too (10% in our case).